decidim-core/lib/decidim/form_builder.rb

```diff
@@ -105,6 +105,68 @@ def categories_select(name, collection, options = {})
 
     private
 
+    def field(attribute, options, html_options = nil)
+      custom_label(attribute, options[:label], options[:label_options]) do
+        class_options = html_options || options
+
+        if error?(attribute)
+          class_options[:class] = class_options[:class].to_s
+          class_options[:class] += " is-invalid-input"
+        end
+
+        options.delete(:label)
+        options.delete(:label_options)
+        help_text = options.delete(:help_text)
+        prefix = options.delete(:prefix)
+        postfix = options.delete(:postfix)
+
+        options[:required] ||= attribute_required?(attribute)
+        max_length = options.delete(:maxlength) || max_length_for_attribute(attribute)
+        options[:pattern] = "^(.){0,#{max_length}}$" if max_length.to_i.positive?
+
+        abide_enabled = options[:pattern] || options[:required]
+
+        content = yield(class_options)
+        content += abide_error_element(attribute) if abide_enabled
+        content = content.html_safe
+
+        html = wrap_prefix_and_postfix(content, prefix, postfix)
+        html + error_and_help_text(attribute, options.merge(help_text: help_text))
+      end
+    end
+
+    def attribute_required?(attribute)
+      object._validators[attribute].any? { |validator| validator.class == ActiveModel::Validations::PresenceValidator }
+    end
+
+    def max_length_for_attribute(attribute)
+      length_validator = object._validators[attribute].find { |validator| validator.class == ActiveModel::Validations::LengthValidator }
+      return unless length_validator
+
+      length_validator.options[:maximum]
+    end
+
+    def custom_label(attribute, text, options)
+      return block_given? ? yield.html_safe : "".html_safe if text == false
+
+      text = default_label_text(object, attribute) if text.nil? || text == true
+      text = safe_join([text.html_safe, yield]) if block_given?
+      label(attribute, text, options || {})
+    end
+
+    def abide_error_element(attribute)
+      defaults = []
+      defaults << :"decidim.forms.errors.#{object.class.model_name.i18n_key}.#{attribute}"
+      defaults << :"decidim.forms.errors.#{attribute}"
+      defaults << :"forms.errors.#{attribute}"
+      defaults << :"decidim.forms.errors.error"
+
+      options = { count: 1, default: defaults }
+
+      text = I18n.t(defaults.shift, options)
```

Comment:
This is intended, it's how `human_attribute_name` does it. `I18n.t` needs a key to translate (thus the shift) and you can give it an array of defaults that it will keep trying until a key exists.